svelte-kit-scss/src/lib/components/PRIssueCard/PRIssueCard.svelte

```diff
@@ -0,0 +1,173 @@
+<script lang="ts">
+  import type { RepoIssue } from '$lib/interfaces';
+  import { relativeTimeFmt } from '$lib/helpers/index.js';
+  import { CheckCircle16, Comment16, NoEntry16 } from 'svelte-octicons';
+
+  export let issue: RepoIssue;
+
+  function isOpen(): boolean {
+    return issue?.state === 'open';
+  }
+
+  function colorMap(color: string): string {
+    switch (color) {
+      case 'D4C5F9': // violet
+        return '#ddd6fe';
+      case '310D85': // dark purple
+        return '#c084fc';
+      case '6ED842': // bright yellow green
+        return '#bef264';
+      case 'ED1DB5': // hot pink
+        return '#f9a8d4';
+      default:
+        return `#${color}`;
+    }
+  }
+</script>
+
+{#if issue}
+  <div class="item">
+    <div class="item-container">
+      <div class="info">
+        {#if isOpen()}
+          <span class="icon {issue.state}">
+            <NoEntry16 />
+          </span>
+        {:else}
+          <span class="icon {issue.state}">
+            <CheckCircle16 />
+          </span>
+        {/if}
+        <div>
+          <div class="content">
+            <span class="title">{issue.title}</span>
+            {#each issue.labels as label}
+              <span class="inline">
+                <span style="background-color: {colorMap(label.color)}" class="label">
+                  {label.name}
+                </span>
+              </span>
+            {/each}
+          </div>
+          <div class="meta">
+            #{issue.number}
+            {#if isOpen()}
+              <span>
+                opened
+                {relativeTimeFmt(issue.createdAt)}
```

Comment:
![image](https://user-images.githubusercontent.com/15368874/199733305-5bb8837f-f35f-45db-82c1-306aed73c4f8.png)

You can fix this with string casting